pkg/kubernetes/scripts/projects.js

```diff
@@ -155,6 +161,52 @@
         }
     ])
 
+    .controller('GroupCtrl', [
+        '$scope',
+        '$routeParams',
+        '$location',
+        'kubeSelect',
+        'kubeLoader',
+        'projectData',
+        'projectActions',
+        'roleActions',
+        'ListingState',
+        function($scope, $routeParams, $location, select, loader, projectData, projectAction, roleActions, ListingState) {
+            loader.watch("users");
+            loader.watch("groups");
+            var group = $routeParams["group"] || "";
+            if (group) {
+
+                var groupObj = select().kind("Group").name(group);
+                if(!groupObj || groupObj.length < 1) {
+                    $location.path("/projects");
```

Comment:
Redirecting when not found isn't a good user experience. But Subin and I discussed that we can fix this in a separate pull request.